src/main/java/com/groupbyinc/api/Query.java

```diff
@@ -1363,6 +1365,33 @@ public Query addBias(String name, String content, Bias.Strength strength) {
   }
 
   /**
+   * @return The session ID
+   */
+  public String getSessionId() {
+    return sessionId;
+  }
+
+  /**
+   * <code>
+   *  A unique string identifier of an end customer. The sessionID should be a unique value for a given user that persists for as
```

Comment:
A unique string identifier of the session that your customer is currently in. The sessionID should be a unique value for a given user that persists for them as long as that user is active on the site for that session. 

We define a session as a the duration of time that you are would normally consider a session of an A/B test. In the future, the traffic split will leverage this information to group a customer into A/B groupings of experiences provided to ensure that the experience is consistent through their sessions.

Therefore, the sessionID should update only if the user is inactive for some period - we recommend keeping this in alignment for what you consider a shopping session for your customers. 

Session ID should not change when the user logs in and can be used to track a user changing from anonymous to logged in. 

Session ID must also be consistent between the Search and Recommendations APIs to ensure correct monitoring of conversion metrics.